.github/workflows/keep_alive.yaml

```diff
@@ -0,0 +1,16 @@
+name: Keep alive the workflow for this entire repository
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+  workflow_dispatch:
+
+permissions:
+  actions: write
+
+jobs:
+  keep_alive:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: gautamkrishnar/keepalive-workflow@v2
```

Comment:
I believe you should use the [`workflow_files`](https://github.com/gautamkrishnar/keepalive-workflow?tab=readme-ov-file#for-github-action) option, set to `monthly-builds.yaml`, to keep the monthly workflow active.